sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/init.sql

```diff
@@ -129,16 +132,17 @@ metrics_ping AS (
         submission_timestamp DESC
     )[SAFE_OFFSET(0)] AS last_reported_adjust_campaign,
   FROM
-    org_mozilla_firefox.metrics AS org_mozilla_firefox_metrics
+    fenix.metrics AS fenix_metrics
   WHERE
     DATE(submission_timestamp) >= '2019-01-01'
   GROUP BY
-    client_id
+    client_id, normalized_channel
```

Comment:
Question, since the same client_id should' appear in more than one channel, why the grouping by channel in all the subqueries?

Reply:
It looks like the same client_id can appear in more than one channel - [see query](https://sql.telemetry.mozilla.org/queries/93416)

Reply:
Actually we should retrieve it in the first subquery (baseline ping) and not from the other pings.